### PR TITLE
T2891: add documentation for ring-buffer option

### PR DESCRIPTION
**Pull request by nvollmar**
## Change Summary
add documentation for ring-buffer configuration

## Related Task(s)
https://vyos.dev/T2891

## Backport
also applies to older branches

## Checklist:
- [X] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-documentation/blob/current/CONTRIBUTING.md) document

### CODE REVIEW

File: docs/configuration/interfaces/ethernet.rst [hunk 1]
```diff
@@ -295,5 +303,3 @@ Operation
         BR margin, min          : 0%
         Vendor SN               : FNS092xxxxx
         Date code               : 0506xx
-
-.. stop_vyoslinter
```
Comment (c-po) [inline docs/configuration/interfaces/ethernet.rst:299]: Linter is started in Line 230 - why remove it here?

**Review by c-po — CHANGES_REQUESTED** [review]

File: docs/configuration/interfaces/ethernet.rst (hunk 1 above)
Comment (rebortg) [reply docs/configuration/interfaces/ethernet.rst:299]: this is fine, linter always run and must start after stop.

**Comment by rebortg** [comment]
@mergifyio backport sagitta

**Comment by rebortg** [comment]
@mergifyio backport equuleus

**Pull request merged**